components/compiler-families/llvm-compilers/SPECS/llvm-compilers.spec

```diff
@@ -0,0 +1,191 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+
+%global gnuver 6.3.0
```

Comment:
Smells like gcc building process (which takes ages), one thing that I like about LLVM is that I don't need to do this multi-stage builds.
Besides, gcc still produces better code than LLVM.
I'm opting to use OHPC gnu-compilers for building LLVM for now and leave comment that we need to drop it later, when all the targeted Linux distros start to use more recent gcc (e.g. 5.3+).